src/main/java/ch/securify/Main.java

```diff
@@ -74,17 +73,21 @@
         @Parameter(names = {"--decompoutputfile"}, description = "output file for the decompiled code")
         private String decompoutputfile;
 
-        @Parameter(names = {"--progress"}, description = "show progress when contracts are being processed")
-        private boolean progress;
+        @Parameter(names = {"-v", "--verbose"}, description = "provide verbose output")
+        private boolean verbose;
 
-        @Parameter(names = {"--pretty"}, description = "display pretty 'clang style' output")
-        private boolean prettyOutput;
+        @Parameter(names = {"-q", "--quiet"}, description = "suppress most output")
+        private boolean quiet;
+
+        @Parameter(names = {"--json"}, description = "provide JSON output to console")
+        private boolean jsonOutput;
     }
 
     private static List<AbstractPattern> patterns;
     private static ContractResult contractResult;
     private static boolean DEBUG = false;
     private static PrintStream log = DEBUG ? System.out : new DevNullPrintStream();
```

Comment:
Given that this is set with the verbose argument, shouldn't the `DEBUG` variable be removed?

Reply:
Yes I left this in case one needs to do a lot of debugging, then it might be practical. I'll remove it.